src/ex5/server.js

```diff
@@ -1,26 +1,20 @@
+const express = require('express')
```

Comment:
be consistent with semicolons ; at the end of each line
it's best practice to always finish a statement with a semicolon

Reply:
you could also configure a linter to warn you when you miss it (and even do it for you)